src/pages/Admin/Admin.tsx

```diff
@@ -277,7 +277,7 @@ const VenueInfoComponent: React.FC<VenueInfoComponentProps> = ({
 
   // @debt Refactor this mapping/customisation into settings, or types/templates, or similar?
   const deleteText =
-    venue.template === VenueTemplate.themecamp ? "Delete camp" : "Delete venue";
+    venue.template === VenueTemplate.themecamp ? "Delete camp" : "Delete space";
```

Comment:
We can also get rid of the "Delete camp" part of this, as it relates to a legacy/irrelevant feature.

```suggestion
    "Delete space";
```

Reply:
This should also be applied to `visitText`, `editText`, etc above.

Since they will just be static strings after this, it may not make sense to keep them as `const`s (assuming they are only used in a single place), so they can probably be inlined into the JSX below.

Reply:
This sort of relates to https://github.com/sparkletown/internal-sparkle-issues/issues/711